packages/editor/src/store/actions.js

```diff
@@ -167,11 +169,24 @@ export function* setupEditor( post, edits, template ) {
 		content = post.content.raw;
 	}
 
-	let blocks = parse( content );
+	let blocks = ( edits && edits.blocks ) || parse( content );
```

Comment:
`edits` is the accumulation of post property values we plan to send as updates to the server.

With your revision here, can you define what purpose `edits` serves?